app/helpers/messages_helper.rb

```diff
@@ -5,36 +5,31 @@
 ###
 module MessagesHelper
   def link_to_edit_message(message)
-    link_to ' Edit message',
+    link_to content_tag(:i, nil, class: 'bi bi-pencil') + content_tag(:span, ' Edit message'),
```

Comment:
I think this tag helper syntax is preferred over `content_tag`:

```suggestion
    link_to tag.i(class: 'bi bi-pencil') + tag.span(' Edit message'),
```